modules/core/arrow-core-data/src/test/kotlin/arrow/core/OptionTest.kt

```diff
@@ -49,7 +50,8 @@ class OptionTest : UnitSpec() {
       TraverseFilterLaws.laws(Option.traverseFilter(), Option.applicative(), ::Some, Eq.any()),
       MonadFilterLaws.laws(Option.monadFilter(), ::Some, Eq.any()),
       HashLaws.laws(Option.hash(Int.hash()), Option.eq(Int.eq())) { it.some() },
-      SemigroupalLaws.laws(Option.semigroupal(), ::Some, ::bijection, associativeSemigroupalEq)
+      SemigroupalLaws.laws(Option.semigroupal(), ::Some, ::bijection, associativeSemigroupalEq),
+      SelectiveLaws.laws(Option.selective(), Eq.any())
```

Comment:
This is not necessary as MonadFilterLaws already check SelectiveLaws. Same for the others.

Reply:
Cool, will remove. In that case I should remove all selective instances if there is a monad instance available too